packages/lib/src/components/Contact/components/ContactPersonCard.module.css

```diff
@@ -0,0 +1,17 @@
+.contact-person-card__card {
+  paddig: var(--mantine-spacing-lg);
+}
+
+.contact-person-card__avatar {
+  margin-top: var(--mantine-spacing-xs);
+  color: indigo;
```

Comment:
Maybe we should use a Mantine variable since the built-in indigo could differ from the Mantine indigo.